bos-distrib/applications/bonita-application-directory.xml

```diff
@@ -5,7 +5,7 @@
         <description>${description}</description>
         <iconPath>bonita-application-directory.png</iconPath>
         <applicationPages>
-            <applicationPage customPage="custompage_ApplicationDirectoryBonitaV6" token="home"/>
+            <applicationPage customPage="custompage_ApplicationDirectoryBonitaV7" token="home"/>
```

Comment:
Why V7 and not V8 here ? V8 are the latest version on master branches for this custom_page

Reply:
You're right, that's an error on my side